plutus-ledger-api/src/PlutusLedgerApi/V3/Contexts.hs

```diff
@@ -261,10 +261,15 @@ newtype ChangedParameters = ChangedParameters {getChangedParameters :: PlutusTx.
     )
 
 data GovernanceAction
-  = ParameterChange (Haskell.Maybe GovernanceActionId) ChangedParameters
+  = ParameterChange
+      (Haskell.Maybe GovernanceActionId)
+      ChangedParameters
+      V2.ScriptHash -- ^ Hash of the constitution script
```

Comment:
Governance policy is optional, in a sense it might not exist. In which case this field will be Nothing
```suggestion
      (Maybe V2.ScriptHash) -- ^ Hash of the constitution script
```